userRoutes/userRoutes.js

```diff
@@ -0,0 +1,26 @@
+const router = require('express').Router()
+
+const Users = require('./helperFunctions')
+
+
+const theWayIsShut = require('../auth/middleware')
+const checkDepartment = require('../auth/checkrole')
+
+router.get('/', theWayIsShut, async (req, res) => {
+    try {
+        const users = await Users.find()
+        let modified = []
```

Comment:
Why did you declare this variable as modified? Would something like verified make more sense?